edc-extensions/cx-policy/src/test/java/org/eclipse/tractusx/edc/policy/cx/usage/ExcludingUsageConstraintFunctionTest.java

```diff
@@ -0,0 +1,63 @@
+/********************************************************************************
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+package org.eclipse.tractusx.edc.policy.cx.usage;
+
+import org.eclipse.edc.participant.spi.ParticipantAgent;
+import org.eclipse.edc.participant.spi.ParticipantAgentPolicyContext;
+import org.eclipse.edc.policy.model.Operator;
+import org.eclipse.tractusx.edc.policy.cx.TestParticipantAgentPolicyContext;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+class ExcludingUsageConstraintFunctionTest {
+
+    private final ParticipantAgent participantAgent = mock();
+    private final ExcludingUsageConstraintFunction<ParticipantAgentPolicyContext> function = new ExcludingUsageConstraintFunction<>();
+    private final ParticipantAgentPolicyContext context = new TestParticipantAgentPolicyContext(participantAgent);
+
+    @Test
+    void evaluate() {
+        assertThat(function.evaluate(Operator.EQ, "x.exclusiveUsage.dataConsumer:1", null, context)).isTrue();
```

Comment:
```suggestion
        assertThat(function.evaluate(Operator.EQ, "cx.exclusiveUsage.dataConsumer:1", null, context)).isTrue();
```

same for other cases in this class

Reply:
Changed that and fixed all the assertions in this class in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/2091/commits/93b154b94f4d53f043e75087c262acd162a177f6).